concourse-webhook-validator/src/concourse_webhook_validator/__init__.py

```diff
@@ -180,16 +182,8 @@ def display_results(webhooks_dict: dict) -> int:
     Table header includes some padding to allow the text to be properly centered
     without being cramped agaist the '|' separators.
     """
-    print(
-        f"{resource_name_text.ljust(int(resource_name_length + 1))}|"
-        + f"{resource_token_valid_text.center(int(len(resource_token_valid_text) + 2))}|"
-        + f"{create_token_matches_text.center(int(len(create_token_matches_text) + 2))}|"
-        + f"{delete_token_matches_text.center(int(len(delete_token_matches_text) + 2))}\n"
-        + f"{'-' * int(resource_name_length + 1)}|"
-        + f"{'-' * int(len(resource_token_valid_text) + 2)}|"
-        + f"{'-' * int(len(create_token_matches_text) + 2)}|"
-        + f"{'-' * int(len(delete_token_matches_text) + 2)}"
-    )
+    webhooks_table = PrettyTable()
```

Comment:
Can the named table header variables above be removed now?

Reply:
Removed in [b272bd4](https://github.com/companieshouse/platform-tools-python/pull/9/commits/b272bd4c9fb6d351cc2a61692619c8ed072d99e9)